chi/lease.py

```diff
@@ -592,14 +594,23 @@ def add_network_reservation(
     if vswitch_name:
         desc_parts.append(f"VSwitchName={vswitch_name}")
 
+    if physical_network != DEFAULT_PHYSICAL_NETWORK:
+        resource_properties = json.dumps(
+            [
+                "and",
+                ["==", "$physical_network", physical_network],
+                ["!=", "$usage_type", "storage"]
```

Comment:
This will only match resources that have `usage_type` set in the first place, and so I think we will run into the same issue @msherman64  mentioned on other sites.